python/dgl/transforms/functional.py

```diff
@@ -3650,44 +3650,69 @@ def random_walk_pe(g, k, eweight_name=None):
 
     return PE
 
-def laplacian_pe(g, k):
+def laplacian_pe(g, k, padding=False, return_eigval=False):
     r"""Laplacian Positional Encoding, as introduced in
     `Benchmarking Graph Neural Networks
     <https://arxiv.org/abs/2003.00982>`__
 
     This function computes the laplacian positional encodings as the
-    k smallest non-trivial eigenvectors (k << n). k and n are the positional
-    encoding dimensions and the number of nodes in the given graph.
+    k smallest non-trivial eigenvectors.
 
     Parameters
     ----------
     g : DGLGraph
         The input graph. Must be homogeneous.
     k : int
-        Number of smallest non-trivial eigenvectors to use for positional encoding
-        (smaller than the number of nodes).
+        Number of smallest non-trivial eigenvectors to use for positional encoding.
+    padding : bool
+        If padding=='false', raise exception when k>=n.
+        Else return (n-1) laplacian positional encodings and (k-n+1) zero encodings
+        (padding) when k>=n.
+        n is the number of nodes in the given graph.
+    return_eigval : bool
+        If return_eigval=='True', return laplacian eigenvalues together with eigenvectors.
+        Else return laplacian eigenvectors only.
```

Comment:
```suggestion
        If True, return laplacian eigenvalues together with eigenvectors.
        Otherwise, return laplacian eigenvectors only. Default: False.
```